lib/inspec/utils/run_data_filters.rb

```diff
@@ -36,7 +37,15 @@ def apply_report_resize_options
         end
       end
 
-      # Find any inputs with :sensitive = true and replace their values with "***"
+      # Filters profiles from report which don't have controls in it.
+      def filter_empty_profiles
+        runtime_config = @config[:runtime_config]
+        if runtime_config[:filter_empty_profiles] && @run_data[:profiles].count > 1
+          @run_data[:profiles].delete_if { |p| p[:controls].empty? }
+        end
+      end
+
+      # Find any inputs with :redact_sensitive_inputsitive = true and replace their values with "***"
```

Comment:
This comment line looks to have gotten mangled... "inputsitive"

Reply:
Fixed.